src/sentry/utils/cursors.py

```diff
@@ -14,8 +14,6 @@
 
 class Cursor(object):
     def __init__(self, value, offset=0, is_prev=False, has_results=None):
-        # XXX: ceil is not entirely correct here, but it's a simple hack
-        # that solves most problems
```

Comment:
I removed this since I have no idea what it means or how it relates to the code below